src/controllers/users-controller.js

```diff
@@ -73,15 +73,15 @@ module.exports = {
 
     ctx.assert(
       user,
-      422,
+      401,
```

Comment:
the same for conditions of missing user/invalid password

[here](https://github.com/gothinkster/node-express-realworld-example-app/blob/ba04b70c31af81ca7935096740a6e083563b3a4a/routes/api/users.js#L58) 422 is returned

I think it is logically correct. As you get this error, to let the user know, that something is wrong with the input and display a validation somehow.

Reply:
Please see https://github.com/gothinkster/koa-knex-realworld-example/pull/12#issuecomment-513127294